### PR TITLE
Add DAG to backfill 3 days of ga_sessions each day

### PR DESCRIPTION
**Pull request by fbertsch**

### CODE REVIEW

File: dags/ga_session_backfill.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+            task_id="checks__fail_" + task_id,
+            source_table="ga_sessions_v1",
+            dataset_id="mozilla_org_derived",
+            project_id="moz-fx-data-shared-prod",
+            is_dq_check_fail=True,
+            owner="frank@mozilla.com",
+            email=["frank@mozilla.com", "telemetry-alerts@mozilla.com"],
+            depends_on_past=False,
+            parameters=[
+                "session_date:DATE:{{ macros.ds_add(ds, " + day_offset + ") }}"
```
Comment (ANich) [inline dags/ga_session_backfill.py:57]: Curious, why isn't `'%Y-%m-%d', '%Y%m%d')` needed here?

Comment (ANich) [reply dags/ga_session_backfill.py:57]: nit: seems like `"macros.ds_add(ds, " + day_offset + ")"` is used in a couple places, you could extract this into a variable above

**Review by ANich — APPROVED** [review]

File: dags/ga_session_backfill.py (hunk 1 above)
Comment (fbertsch) [reply dags/ga_session_backfill.py:57]: This one is the BQ parameter, which expects the dash-separated strings (e.g. `2023-11-01`).

Comment (fbertsch) [reply dags/ga_session_backfill.py]: Much more readable, thank you for the suggestion!

Comment (fbertsch) [reply dags/ga_session_backfill.py]: Oh I see why you asked! The other parameter was wrong, this one was right. I changed the other one to include dashes, thank you!

**Pull request merged**